### ISSUE TITLE
Update code sample for array fields in "About the `schema` and `uiSchema` objects"

### ISSUE THREAD

**Issue by bernars-usa** [usds/us-forms-system#172]
From @annekainicUSDS in https://github.com/usds/us-forms-system/pull/167/files#r203389122:

> The only thing I don't like about this code example is that we're introducing new data, branchName and toursOfDuty, which aren't included in the other examples, so it's a bit confusing. If possible I think it's helpful when including code examples to stick with a particular theme or set of code and iterate on that.

Here's the code sample, for reference:

```js
{
  'ui:title': 'My form',
  toursOfDuty: {
    items: {
      branchName: {
        'ui:title': 'Branch'
      }
    }
  }
}
```

If someone wants to comment with a better code sample here, I can implement.

**Comment by jcmeloni-usds** [comment]
@bernars-usa Do we still need a new code example? Say the word and I can do something.

**Comment by bernars-usa** [comment]
Yeah, `toursOfDuty` is still used in https://github.com/usds/us-forms-system/blob/master/docs/building-a-form/about-the-schema-and-uischema-objects.md#understanding-the-uischema-object.

**Comment by jcmeloni-usds** [comment]
Addressed in #305
